h/templates/app.html.jinja2

```diff
@@ -52,6 +52,25 @@
       <main ng-view=""></main>
     </div>
 
+    <div class="loggedout-message" ng-if="isSidebar && auth.status === 'signed-out' && settings.annotations" ng-cloak>
+      This is a public annotation created with Hypothes.is
```

Comment:
Ah, I've realized that this might not be a public annotation. It could also be either a private annotation or a group annotation. As it currently stands, we'll only know that once the annotation finishes loading.